internal/pkg/docker/docker.go

```diff
@@ -74,6 +96,26 @@ func (r Runner) Push(uri, imageTag string) error {
 	return nil
 }
 
+// PushWithMultipleTags is the same as Push but but is able to accept multiple tags.
+func (r Runner) PushWithMultipleTags(uri string, imageTags []string) error {
+	var errMsg []string
+
+	for _, imageTag := range imageTags {
+		path := imageName(uri, imageTag)
+
+		errPush := r.Run("docker", []string{"push", path})
+		if errPush != nil {
+			errMsg = append(errMsg, fmt.Sprintf("push tag %s: %v", imageTag, errPush))
+		}
```

Comment:
Can you tell us again the use case for this method? I'm trying to decide if it's better to just fail fast and immediate err on the first failure instead of continuing with more pushes.

Reply:
For example, we may want to push an image both with a user specified tag and a "latest" tag. It makes sense to not continue with more pushes. Since it is able to `Build`, it indicates that the tags' names are fine. If it ever fails, then it's not as likely that the following pushes will succeed